content/reference/api/hub/latest.yaml

```diff
@@ -118,15 +118,19 @@ paths:
     post:
       tags:
         - authentication
-      summary: Create an authentication token
+      summary: Create an authenticated token
```

Comment:
I prefer the previous version; it is for authenticating to Docker Hub.

Reply:
> I prefer the previous version; it is for authenticating to Docker Hub.

The old version is slightly wrong. We are trying to fix verbiage in many places. This route authenticated you. There is no more authentication to be done.

Reply:
If it helps, this route is gonna be deprecated in the mid-future.

Reply:
I'm going to just keep it as is... after thinking a bit, we a lot of work to do on this as an auth team and new APIs are coming in to replace.